src/@types/index.ts

```diff
@@ -0,0 +1,92 @@
+export interface FetchResponse {
```

Comment:
Interesting. Didn't realize that would be be the effect. I always wondered how sometimes git is able to mark a file as renamed rather than deleted. I'll try to remember this for the future.